### ISSUE TITLE
SSL_connect failed r:2 error

### ISSUE THREAD

**Issue by preserveddarnell** [nemosminer/DSTM-equihash-miner#13]
zm --server equihash.mine.zpool.ca --port 2142 --user t1fkxCWJs3f2oXznGwZuEgftJ2SCjYZ8VjZ --pass c=ZCL

Whenever I start it up, it always prompts for SSL_connect failed r:2 error: 14094410:SSL routines:ssl3_read_bytes:sslv3 alert handshake failure.

Any solutions?

**Comment by hawk-eye77** [comment]
I have same problem on nanopool from yesterday. Please help!

**Comment by preserveddarnell** [comment]
> I have same problem on nanopool from yesterday. Please help!

So in the meantime I am just using the latest version of EBWF miner since that works without SSL.

**Comment by hawk-eye77** [comment]
does it support ZEC mining, what's --algo setting should I use? didn't find the answer...

**Comment by preserveddarnell** [comment]
Yeah it's just an alternative to DSTM, same Nvidia GPU equihash miner. I think it's the same flags and options as dstm. 

I just use this:

miner 
--server 
 --port 
--i 21 
--user  
--pass
